pkgs/test_core/pubspec.yaml

```diff
@@ -25,7 +25,7 @@ dependencies:
   source_span: ^1.4.0
   stack_trace: ^1.9.0
   stream_channel: ">=1.7.0 <3.0.0"
-  vm_service: '>=1.0.0 <3.0.0'
+  vm_service: '>=3.0.0 <4.0.0'
```

Comment:
Do we need to increase the lower bound here?

Reply:
There's the potential for a non-expected `Sentinel` return value in `package:vm_service` <3.0.0 which will cause a type error. It's unlikely we'll bump into it here, but I wanted to be safe rather than sorry. I'm content with leaving the lower bound as-is if that's what you'd prefer.

Reply:
I think we would prefer to leave it - assuming this was a pre-existing issue? We haven't had reports of it and would prefer to leave this lax so that version solving is more likely to succeed for people.

Reply:
Fair enough. Done.